Vocabularies/Laugh.js

```diff
@@ -0,0 +1,7 @@
+function laughVocabulary() {

```

Comment:
Lol vocab?

Reply:
yeah, could be merged... I think I created it to cover dangling references